cli/compose/loader/loader_test.go

```diff
@@ -1395,3 +1395,57 @@ networks:
 	}
 	assert.DeepEqual(t, config, expected, cmpopts.EquateEmpty())
 }
+
+func TestFilePathsCrossPlat(t *testing.T) {
+	cases := []struct {
+		path       string
+		workingDir string
+		expected   string
+	}{
+		{
+			path:       `/var/data`,
+			workingDir: `/working/dir`,
+			expected:   `/var/data`,
+		},
+		{
+			path:       `/var/data`,
+			workingDir: `c:\working\dir`,
+			expected:   `/var/data`,
+		},
+		{
+			path:       `relative/data`,
+			workingDir: `/working/dir`,
+			expected:   `/working/dir/relative/data`,
+		},
+		{
+			path:       `relative/data`,
+			workingDir: `c:\working\dir`,
+			expected:   `c:\working\dir\relative\data`,
+		},
+		{
+			path:       `c:\var\data`,
+			workingDir: `/working/dir`,
+			expected:   `c:\var\data`,
+		},
+		{
+			path:       `c:\var\data`,
+			workingDir: `c:\working\dir`,
+			expected:   `c:\var\data`,
+		},
+		{
+			path:       `relative\data`,
+			workingDir: `/working/dir`,
+			expected:   `/working/dir/relative/data`,
+		},
+		{
+			path:       `relative\data`,
+			workingDir: `c:\working\dir`,
+			expected:   `c:\working\dir\relative\data`,
+		},
```

Comment:
These tests look great, but we're missing test coverage for the crosspath.HomeRooted case, arent' we?